android/build.gradle

```diff
@@ -79,7 +79,8 @@ android {
 
     defaultConfig {
         versionName project.ext.versionName
-
+        renderscriptTargetApi 16
```

Comment:
raw renderscript does need it but setting this to lower version enables support.v8 version of renderscript (alternatively check version and when >= 17 use renderscript and old blur function otherwise)